design/FY2021/NFP-runarouind-heat-recovery-loop.md

```diff
@@ -0,0 +1,143 @@
+
+The following figures and text were provided in the Trane TRACE 700 User’s Manual • CDS-PRM001-EN at <https://software.trane.com/RightNow/0103-CoilLoopEnergyRecovery/Coil_loop_exhaust-air_energy_recovery.pdf>. 
+
+A simple schematic of a run-around coil heat recovery loop is shown in Figure 1. Insert A and B show heat recovery between two air stream while insert C shows energy transfer from before to after a cooling coil (similar to `CoilSystem:Cooling:DX:HeatExchangerAssisted`. 
+
+![RunaroundCoilLoop](RunaroundCoilHXLoop.png)
+## *Figure 1. Run around coil loop schematic.* ##
+
+Figure 2 shows an example run-around coil design for winter and summer operation. During the heating season (Inset A), heat extracted from the exhaust air stream (EA)
+warms the air brought into the building. Operation of the coil loop is limited to prevent the supply-air temperature from exceeding the cooling set point. (This condition is most likely to occur on mild days during the spring and fall.) Preconditioning the outdoor
+air (OA) in this manner reduces the heating load, which in turn reduces the energy consumption of the HVAC system.
+
+![RunaroundCoilDesignExample](RunaroundCoilDesignExample.png) 
+## *Figure 2. Run around coil design example.* ##
+
+## Approach ##
+
+There are 3 water coil models in EnergyPlus.
+
+1. Coil:Cooling:Water
+2. Coil:Cooling:Water:DetailedGeometry
+3. Coil:Heating:Water
+
+As shown in figure 2, one coil heats in winter and cools in summer while the other does the reverse as is typical in heat recovery components. For this reason, a water coil model that can exchange heat based on the entering air and water temperature is required. It is believed that the cooling coil model behaves this way (i.e., can heat the air under certain conditions). I have also seen this a few times and MJWitte also believes the water cooling coil model has this capability.
+
+There are two possible approaches I can think of. 
+
+1. Connect 2 water cooling coils in a typical plant configuration. With one coil on the demand side of the plant loop while the other is a supply side component. This would allow alternate configurations if desired. For example to allow a boiler, chiller, evap cooler, etc. on the supply side which maintains a certain temperature to the supply side coil. Although I can't think of a reason to do this, it seems like a very flexible way to achieve the goal for a run-around coil application. A user could create a plant loop and then add two coils and a pump. 
+
+**Pro:** This approach may require minimal changes to the IDD (the PlantLoop field for Plant Equipment Operation Scheme Name is required). Would provide flexibility in configuration. Pipe losses could be modeled.  
+**Con:** The coil model would use ControlcompOutput as the solution algorithm. The air-side and water-side calculations would be disassociated the same as water coils on a branch are currently modeled.
```

Comment:
Why would it need to use ControlCompOutput? Seems like a versatile coil controller that can switch between heating and cooling is the main thing to do here and it could use root solver. 

Plant loop approach can also provide autosizing, support for different glycol mixtures, different pumping arrangements, pump heat to fluid modeling, and ability to combine multiple air handlers.